From 97410031ab920c8fde38a2d489e4d8a4efa0a7c8 Mon Sep 17 00:00:00 2001
From: Wenyou Yang <wenyou.yang@atmel.com>
Date: Mon, 19 Jan 2015 09:42:55 +0800
Subject: [PATCH] board: config: sama5d4: add BUS_SPEED=148/170MHz support

SUPPORT_BUS_SPEED_148MHZ/SUPPORT_BUS_SPEED_170MHZ are missed,
so add 148/170MHz support in the menuconfig.

Signed-off-by: Wenyou Yang <wenyou.yang@atmel.com>
---
 board/sama5d4_xplained/Config.in.board | 2 ++
 board/sama5d4ek/Config.in.board        | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/board/sama5d4_xplained/Config.in.board b/board/sama5d4_xplained/Config.in.board
index 8944605..56143ff 100644
--- a/board/sama5d4_xplained/Config.in.board
+++ b/board/sama5d4_xplained/Config.in.board
@@ -20,5 +20,7 @@ config CONFIG_SAMA5D4_XPLAINED
 	select CONFIG_HAS_PMIC_ACT8865
 	select CONFIG_PM
 	select SUPPORT_BUS_SPEED_176MHZ
+	select SUPPORT_BUS_SPEED_170MHZ
+	select SUPPORT_BUS_SPEED_148MHZ
 	help
 	  Use the SAMA5D4_Xplained Development board
diff --git a/board/sama5d4ek/Config.in.board b/board/sama5d4ek/Config.in.board
index 4611bcf..55a60a0 100644
--- a/board/sama5d4ek/Config.in.board
+++ b/board/sama5d4ek/Config.in.board
@@ -20,5 +20,7 @@ config CONFIG_SAMA5D4EK
 	select CONFIG_HAS_PMIC_ACT8865
 	select CONFIG_PM
 	select SUPPORT_BUS_SPEED_176MHZ
+	select SUPPORT_BUS_SPEED_170MHZ
+	select SUPPORT_BUS_SPEED_148MHZ
 	help
 	  Use the SAMA5D4EK Development board
-- 
1.9.1

